includes/Core/CLI/Reset_CLI_Command.php

```diff
@@ -27,16 +28,29 @@ class Reset_CLI_Command extends CLI_Command {
 	 *
 	 * ## OPTIONS
 	 *
+	 * [--persistent]
+	 * : Remove persistent site kit options too.
```

Comment:
Slight tweak of the description for the CLI flag.
```suggestion
	 * : Additionally deletes persistent options.
```